### DISCUSSION TITLE
Failed to run `config` for catppuccin

### DISCUSSION THREAD

**Discussion by SimFG** [ayamir/nvimdots#870] [Q&A] [answered]
![image](https://github.com/ayamir/nvimdots/assets/21985684/825f65c7-d8b6-409a-9ca4-f862e931f6a5)

use the latest master source

**Comment by charliie-dev** [comment, marked as answer]
`:Lazy sync`

**Comment by SimFG** [comment]
it can't work. when i use the `nvim ./`, there is a error like below
![image](https://github.com/ayamir/nvimdots/assets/21985684/d3deaae6-8009-4805-ba21-aa8e3ec31aac)

**Reply by charliie-dev** [reply]
Plugin installation failed due to networking issues. Remove the plugin folder and reopen nvim.

If u have further questions, check past issues and repo wiki first.

**Reply by SimFG** [reply]
thank you so much
